lib/coach/handler.rb

```diff
@@ -25,16 +25,28 @@ def call(env)
       start = Time.now
 
       publish('coach.handler.start', start_event.dup)
-      response = chain.instrument.call
 
-      finish = Time.now
-      publish('coach.handler.finish',
-              start, finish, nil,
-              start_event.
-                merge(response: { status: response[0] },
-                      metadata: context.fetch(:_metadata, {})))
+      begin
+        response = chain.instrument.call
+      rescue => e
+        finish = Time.now
+        publish('coach.handler.error',
+                start, finish, nil,
+                start_event.merge(
+                  response: { status: 500 },
```

Comment:
This is my major concern. At this point, a response has not actually been calculated (it's `nil` because of the exception) - so this isn't a true status. However, I suspect other tools might be relying on this attribute. We've got some options:

1. Use a status which could only come from Coach
2. Send a status string like "coach_error"
3. Don't send this `response` object at all.

Thoughts?